### PR TITLE
fix(docs): modified grammar and phrasing in docs and comments

### PR DESCRIPTION
**Pull request by ejcheng**
Checklist:

- [X] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [X] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [X] My pull request targets the `main` branch of freeCodeCamp.
- [X] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Comment by ieahleen** [comment]
Thank you for opening this pull request.

This is a standard message notifying you that we've reviewed your pull request and have decided not to merge it. We would welcome future pull requests from you.

Thank you and happy coding.

**Comment by nhcarrigan** [comment]
Heya @import-brain,

Thanks for your interest in contributing!

In order for us to be able to review this pull request, we would need you to resolve the linting issues indicated here:
image


If you have any questions about this process, you can [review our contributing guidelines](https://contribute.freecodecamp.org) to run the linter locally, or [join our contributor chat room](https://chat.freecodecamp.org/channel/contributors) to ask for assistance. 🙂

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

File: client/src/utils/index.ts [hunk 1]
```diff
@@ -1,6 +1,9 @@
...
-// before we try to validate
+/*
+ * This regex is not for validation, it is purely to see
+ * if we are looking at something like what we want to validate
+ * before we try to validate.
+ */
 export const maybeEmailRE = /.*@.*\.\w\w/;
 export const maybeUrlRE = /https?:\/\/.*\..*/;
 export const hasProtocolRE = /^http/;
+
```
Comment (nhcarrigan) [inline client/src/utils/index.ts:9]: Just need to remove this extra blank line 🙂

Comment (import-brain) [reply client/src/utils/index.ts]: Just did, I squashed the commits and force pushed to my fork, waiting for tests to work

Comment (import-brain) [reply client/src/utils/index.ts]: @nhcarrigan Sorry for my "newbieness", I'm very new to linters, so I made some mistakes

Comment (nhcarrigan) [reply client/src/utils/index.ts]: Happy to help! Looks like you've resolved the lint issues. Nice work 😁

**Review by naomi-lgbt — APPROVED** [review]
This is looking good to me!

**Comment by import-brain** [comment]
> 
> 
> This is looking good to me!

Yay! :)

**Review by ShaunSHamilton — APPROVED** [review]
This LGTM 👍 

---

Please be aware your `main` branch will need a reset, and we encourage you to work off of a new branch, for future PRs: https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally

**Comment by ShaunSHamilton** [comment]
Congrats on your first pull request (PR)! 🎉

Thank you, for your contribution to the page! 👍
We are happy to accept these changes, and look forward to future contributions. 📝

**Comment by import-brain** [comment]
> 
> 
> This LGTM 👍
> 
> Please be aware your `main` branch will need a reset, and we encourage you to work off of a new branch, for future PRs: https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally

Thanks! I'll use a new branch for future PRs

**Pull request merged**